mesa/visualization/components/matplotlib.py

```diff
@@ -618,30 +618,46 @@ def _scatter(ax: Axes, arguments):
             )
 
 
-def make_plot_measure(measure: str | dict[str, str] | list[str] | tuple[str]):
```

Comment:
Letting this function call a warning / informative error is an oneliner right?

Reply:
and more lines to move around in some of the other refactoring still to come to address other open issues.

Reply:
I just checked, None of this is in 2.4 only in the mesa 3 alpha's and beta's. So I really don't see the point of bothering with warnings. I am going to move more stuff around for example for #2434. I agree that warnings etc. are in order for official releases and when deprecating stable stuff. This has never been officially stable yet anyway.

Reply:
I don’t like such breaking changes without proper error handling so late in the beta release cycle, but since we never released this as stable before I find it acceptable if it’s your strong preference.

Don’t expect me to make an habit out of it ;)

<hr>

If I knew we were going to do such extensive overhauls I would have suggested staying longer in alpha, maybe something to more explicitly discuss if/when 4.0 is targeted.

Reply:
I did not know that either until I started looking at this stuff. 

Do you have any sense of how many downloads there have been for some of the pre releases?

Reply:
Also, we were more or less in agreement that the components part of solara would be experimental in 3.0 anyway 😄